### ISSUE TITLE
Normalization Layers

### ISSUE THREAD

**Issue by NitzanHod** [NitzanHod/deep-night#35]
Using the added model configuration manager add normalization layers to the models.py module.
You should add the following:
1. Batch Normalization
2. Instance Normalization
3. Layer Normalization
4. Group Normalization - with configurable group number 'groups'

Note: the main challenge in this issue may be handling the separation of activation functions from previous convolution layers, in order to allow combinations such as conv-batchnorm-relu.

Note 2: In the future this class should be wrapped with a GUI.
